package.json

```diff
@@ -67,9 +70,11 @@
     "@typescript-eslint/parser": "^7.11.0",
     "@vitest/coverage-v8": "1.6.0",
     "auto-changelog": "^2.4.0",
+    "bullmq": "^5.8.4",
```

Comment:
why have them as both runtime and dev dependencies?

Reply:
one is `peer` another is `dev` for types and stuff

I can move to `dependencies` if that's fine for you - I think `bullmq-pro` depends on the basic version anyways

Reply:
ah, gotcha, GH didn't show me the peerDependencies part.

you are right, bullmq-pro does depend on the bullmq.

Either approach is fine, can be peerDependencies or prod